src/widgets/programs/programs-form.component.tsx

```diff
@@ -59,15 +60,18 @@ export default function ProgramsForm(props: ProgramsFormProps) {
 
   useEffect(() => {
     if (patientUuid) {
-      const sub1 = fetchLocations().subscribe(locations => {
-        return setLocations(locations);
-      }, createErrorHandler());
+      const sub1 = fetchLocations().subscribe(
+        (locations: Array<LocationData>) => {
```

Comment:
Why do we need to specify the type? `fetchLocations` should already come with an observable of `Array<LocationData>`.